### DISCUSSION TITLE
Exclamation mark / other aliases

### DISCUSSION THREAD

**Discussion by b-van-b** [pit-ray/win-vind#257] [General]
In the usage tutorial, sometimes `:e` and `:!` are used. From the documentation, I was able to figure out that `:e` is an alias for `:execute`, but what is `:!`? It appears to do the same thing as `:e`. Also, are there other default aliases aside from these two?

**Comment by pit-ray** [comment]
The `:e` and `:execute` open the URL or file path in the default software. Just like performing a double-click in Explorer.
https://pit-ray.github.io/win-vind/cheat_sheet/functions/#execute

On the other hand. `:!` is the same as executing a command at the command prompt. For example, executing `:!dir;` will display a list of home directories. (`;` leaves the resulting command prompt).
https://pit-ray.github.io/win-vind/cheat_sheet/functions/#start_external

There are several default aliases defined as shown [here](https://pit-ray.github.io/win-vind/cheat_sheet/defaults/small/#process-launcher), but you can define any command you like by mapping it to execute the corresponding command `<execute>` or `<start_external>` as follows.
```vim
command mycommand <execute>
```

Thanks.
